setup/features/server-group-launch-settings/aws-ec2/launch-templates.md

```diff
@@ -0,0 +1,322 @@
+    </tr>
+    <tr>
+      <td>CreditSpecification</td>
+      <td>The credit option for CPU usage of the instance.
+      Valid for burstable performance EC2 instances only. <font size="-1">Learn more from <a target="_blank" href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/burstable-credits-baseline-concepts.html">AWS</a>.</font></td> 
+      <td><em>enableCpuCredits</em></td>
+      <td><em>unlimitedCpuCredits</em></td>
+      <td><em>v1.24</em></td>
+      <td><a target="_blank" href="https://github.com/spinnaker/clouddriver/blob/master/clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/BasicAmazonDeployHandler.groovy#L488">
+          default <em>false</em></a> used only when all instance types in request support bursting.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Diversified Server Groups with Launch Templates
+Enabling launch templates in Clouddriver also unlocks additional features offered by AWS EC2 AutoScaling that could be helpful for use cases like:
+* create Server Groups with flexible instance configuration e.g. multiple instance types, a combination of purchase options (On-Demand / Spot) in order to tap into multiple Spot Instance pools.
+* use instance weighting to specify the relative weight of each instance type, to count towards the desired capacity of the group.
+* allow AWS to optimize instance allocation by using allocation strategies.
+* optimize by capacity but prioritize the list of instance types provided to indicate preference.
+* maintain a consistent baseline of On-Demand capacity and use Spot in remaining capacity for cost savings.
+* follow [AWS recommended best practices for EC2 Spot](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-best-practices.html) for better user experience.
+
+#### Motivation to diversify instances in your Server Group
+* Reduce probability of [InsufficientInstanceCapacity](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/troubleshooting-launch.html#troubleshooting-launch-capacity) exceptions with flexible instance configuration.
+* Reduce costs by diversifying instances across purchase options and Spot allocation strategies. See AWS docs to learn more about [how to use Spot](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-best-practices.html).
+* Reduce costs and optimize resources with instance weighting.
+* Enhance availability by deploying your application across multiple instance types running in multiple Availability Zones.
+* Maintain desired Spot capacity by proactively augmenting your fleet with a new Spot instance from an optimal pool before a running instance is interrupted by EC2, by enabling [capacity rebalance](https://docs.aws.amazon.com/autoscaling/ec2/userguide/capacity-rebalance.html).
+
+Using one or more parameters in the table below, in Clouddriver API request will automatically create Server Groups with the instance diversification configuration specified and AWS defaults for those not specified.
```

Comment:
Does this make sense? "... specified, and will use AWS defaults for the parameters not specified."